src/api/blockApi.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import helpers from '../utils/helpers';
+
+import requestExplorerServiceV1 from './axiosInstance';
+
+
+const BLOCK_API_DEFAULT_TIMEOUT = 35000; // ms
```

Comment:
Why so high this timeout? What's the API server timeout?

Reply:
the Elastic Search timeout is 25 seconds, so the explorer-services times out on 30 seconds. This is why I put 35 seconds. I do agree is too high, but I do not know if this will impact too much the experience, as this is handle async.